android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -640,6 +652,20 @@ private String encodeChannelEvent(ChannelEvent event) {
     return jsonMap;
   }
 
+  private Map<String, Object> encodeChannelMessageExtras(MessageExtras c) {
```

Comment:
What does `c` and `v` stand for?

Reply:
shorthand references were used in this class for all encoder/decoders to keep the code compact. As it is an internal detail much attention was not given to why that specific character was chosen. They were all just duplicated and modified to suit necessary encoder.